### PR TITLE
task 7 completed

### PR DESCRIPTION
**Pull request by Glt1b**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -110,3 +110,60 @@ describe('GET/api/review/:review_id', () => {
...
+    return request(app)
+      .post('/api/reviews/2/comments')
+      .send(comment)
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toEqual('Bad Request');
+      });
+  });
+});
+
```
Comment (emilyb93) [inline __tests__/app.test.js:176]: We also need a test for when the user is provided but does not exist in the database

File: models/games_models.js [hunk 2]
```diff
@@ -36,5 +36,26 @@ exports.selectReviewById = (review_id) => {
...
+    (author, body, review_id, created_at, votes ) 
+    VALUES 
+    ($1, $2, $3, $4, 0)
+    RETURNING *;`, [body.username, body.body, review_id, new Date().toLocaleDateString()]).then((result) => {
+
+        return result.rows[0];
+    })
+        }
+});
+}};
```
Comment (emilyb93) [inline models/games_models.js]: A few notes here, we dont need to select from reviews to find out if it exists firsts. When we insert the comment with the review_id, this is a foreign key, if they key doesnt exist to make a link in the other table, SQL will throw an error and we can then handle that error.

Also, we dont need to give the created_at and votes fields, these will be added by SQL as they have DEFAULT properties in the seed function when we created the table.

Lastly, this is starting to look a lot like nested promises and i want us to move away from, please refactor so this is only in a single promise chain

**Review by emilyb93 — CHANGES_REQUESTED** [review]
Left a comment

**Review by Harrison365 — CHANGES_REQUESTED** [review]
Hey Marcin.
Please...
-Test for specific review_id (it may be worth testing the whole response rather then separately like you are doing now)
-Test for a username that doesn't exist (404)

Thanks :)

**Review by Harrison365 — CHANGES_REQUESTED** [review]
Hey Marcin :) 
Please...
-Test that the correct comment_id comes back 
-Add test for present user, missing body
-Add test for present username and body but the username doesn't exist.

Thanks :)

**Review by emilyb93 — APPROVED** [review]

**Pull request merged**